README.md

```diff
@@ -1,32 +1,214 @@
+Above variables will also be produced with the output of `facilitator init` command.
+
+**6. Fund facilitator workers for gas and bounty**: Workers created in facilitator init step needs to be funded for gas and bounty on both chains.
+
+   *Gas on origin*: Facilitator needs to pay for gas on origin chain. Origin worker address created in `facilitator init` step must be funded to pay for transaction fee. 
+   
+   There are various ways to fund on origin chain, for testnet any public faucet can be used. Below web3 transaction can also be done, if there is fund in an existing key.
+   
+      
+      web3.eth.sendTransaction(
+      {
+         from:<funder_address>, 
+          to: <origin_worker>,
+          value:<Fund_in_wei
+      })
+      
+   *Gas on auxiliary*: Facilitator also needs to pay for transaction fee on auxiliary chain. For testnet [mosaic faucet](https://github.com/mosaicdao/faucet) can be used to get fund. Alternatively, OST on value chain can also be converted to base token on auxiliary chain in order to pay for gas. This is explained in detail in section [request stake]().
+   
+   *Value token for bounty*: Facilitator needs to stake bounty to gateway/co-gateway in order to perform stake & mint and redeem & unstake. Bounty token on the origin chain for existing mosaic chain is simple token and on the auxiliary chain bounty is base token. 
+   
+   On testnet, bounty can be funded to workers using [mosaic faucet](https://github.com/mosaicdao/faucet).
+   
+**7. White list workers**: origin and auxiliary workers should be whitelisted in stakepool and redeem pool contract respectively. Below commands will whitelist the workers. 
+
+   1. Clone git repository:
+   ```
+       git clone https://github.com/mosaicdao/mosaic-chains.git
+   ```
+   2. Set below enviornment variables
+   
+    
+    export ORIGIN_WEB3_ENDPOINT='replace_with_origin_web3_endpoint';
+    export AUXILIARY_WEB3_ENDPOINT='replace_with_auxiliary_web3_endpoint';
+    export AUXILIARY_CHAIN_ID='replace_with_auxiliary_chain_id';
+    export MOSAIC_CONFIG_PATH='replace_with_mosaic_config_path';
+    export ORIGIN_WORKER_ADDRESS='replace_with_origin_worker_address';
+    export AUXILIARY_WORKER_ADDRESS='replace_with_auxiliary_worker_address';
+    export ORIGIN_WORKER_EXPIRATION_HEIGHT='replace_with_origin_expiration_height';
+    export AUXILIARY_WORKER_EXPIRATION_HEIGHT='replace_with_auxiliary_expiration_height';
+
+origin and auxiliary worker addresses are generated with `facilitator init` step. 
+Mosaic config path for supported chain should be available on `~/.mosaic/<origin-chain>/mosaic.json` where `<origin-chain>` is origin chain identifier e.g. `ropsten`.  
+
+origin and auxiliary worker expiration height is block height from current block for which worker keys are whitelisted. If current block is 1000 and expiration height is set to 100 then worker keys will be whitelisted for 1100 block.
+
+   3. run command:
+     
+     npm run whitelist-workers
```

Comment:
will he run this from facilitator repo?

Reply:
No, step 1 is about cloning mosaic-chains.